### PR TITLE
refactor QDateTime to ISO 8601 with offset string

### PR DESCRIPTION
**Pull request by otahirs**

### CODE REVIEW

File: libquickevent/libquickeventcore/src/utils.cpp [hunk 1]
```diff
@@ -5,26 +5,10 @@
...
-				offset_min = -offset_min;
-			}
-			else {
-				ret += '+';
-			}
-			ret += QStringLiteral("%1:%2").arg(offset_min / (60 * 60), 2, 10, QChar('0')).arg((offset_min / 60) % 60, 2, 10, QChar('0'));
-		}
-	}
-	return ret;
+	dt.setOffsetFromUtc(dt.offsetFromUtc());
```
Comment (fvacek) [inline libquickevent/libquickeventcore/src/utils.cpp:10]: neni toto idempotence?

Comment (otahirs) [reply libquickevent/libquickeventcore/src/utils.cpp:10]: Its not a bug, its a feature. :)

https://bugreports.qt.io/browse/QTBUG-26161?focusedCommentId=554227&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-554227

Tested it does not work - the time zone offset is missing - without this line. (at least as for today).

Comment (fvacek) [reply libquickevent/libquickeventcore/src/utils.cpp:10]: interesting :)

Comment (fvacek) [reply libquickevent/libquickeventcore/src/utils.cpp:10]: please add comment with short explanation of this strange line.

**Pull request merged**